syscallf/zsyscall_linux_386.go

```diff
@@ -0,0 +1,17 @@
+// mksyscall.pl -l32 -tags linux,386 syscall_linux.go syscall_linux_386.go
+// Code generated by the command above; DO NOT EDIT.
+
+//go:build linux && 386
```

Comment:
All this files look similar to me, so what we could do is just have `//go:build linux` and just one implementation

Reply:
You're right, hehe :) I will prepare one after comparison.
Maybe we would not need copy right then, hence we wrap only.